models/public/pelee-coco/model.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+     The Pelee is a Real-Time Object Detection System on Mobile Devices. <Pelee> <https://arxiv.org/pdf/1804.06882.pdf>
+     network trained on MSCOCO dataset. This model is implemented using the Caffe* framework. For details about this model,
```

Comment:
Something went wrong with this description; the sentence that spans the previous line and this one looks broken.

Reply:
sorry for the stupid question, how I can check it? :)

Reply:
What do you mean? Just read it. :)

Reply:
ok., oh, I see...

Reply:
anyway it should be aligned with description